dev-utilities/deprecatedUtilityClasses.js

```diff
@@ -0,0 +1,296 @@
+// List of disallowed utility classes that are exported from Kongponents
+// These classes **must not** be used in this repository
+module.exports = [
+  'type-xxxl',
+  'type-xxl',
+  'type-xl',
+  'type-lg',
+  'type-md',
+  'type-sm',
+  'type-xs',
+  'type-xxs',
+  'mono',
+  'color-blue-100',
+  'color-blue-200',
+  'color-blue-300',
+  'color-blue-400',
+  'color-blue-500',
+  'color-blue-600',
+  'color-blue-700',
+  'color-petrol-100',
+  'color-petrol-200',
+  'color-purple-100',
+  'color-purple-200',
+  'color-purple-300',
+  'color-purple-400',
+  'color-steel-100',
+  'color-steel-200',
+  'color-steel-300',
+  'color-steel-400',
+  'color-steel-500',
+  'color-steel-600',
+  'color-steel-700',
+  'color-red-100',
+  'color-red-200',
+  'color-red-300',
+  'color-red-400',
+  'color-red-500',
+  'color-red-600',
+  'color-red-700',
+  'color-green-100',
+  'color-green-200',
+  'color-green-300',
+  'color-green-400',
+  'color-green-500',
+  'color-green-600',
+  'color-green-700',
+  'color-teal-100',
+  'color-teal-200',
+  'color-teal-300',
+  'color-teal-400',
+  'color-teal-500',
+  'color-yellow-100',
+  'color-yellow-200',
+  'color-yellow-300',
+  'color-yellow-400',
+  'color-yellow-500',
+  'color-yellow-600',
+  'color-grey-100',
+  'color-grey-200',
+  'color-grey-300',
+  'color-grey-400',
+  'color-grey-500',
+  'color-grey-600',
+  'color-black-85',
+  'color-black-70',
+  'color-black-45',
+  'color-black-25',
+  'color-black-10',
+  'color-black-100',
+  'color-black-200',
+  'color-black-300',
+  'color-black-400',
+  'color-black-500',
+  'color-white',
+  'style-heading-1',
+  'style-heading-2',
+  'style-heading-3',
+  'style-heading-4',
+  'style-body-lg',
+  'style-body-lg-bold',
+  'style-body-md',
+  'style-body-md-bold',
+  'style-body-sm',
+  'style-body-sm-bold',
+  'style-body-tiny',
+  'style-body-link',
+  'style-body-bc',
+  'style-body-code',
+
+  /* We define the following classes.
```

Comment:
why is this commented out? should we just remove it then?

Reply:
I wanted to keep the list complete for reference. Once Kongponents is updated to its next major version, these utility classes will be removed. Once we updated Kongponents, we will also remove the added linter and those rules again as there is no benefit in disallowing them then anymore.

In short: this will be short-lived.